tools/bin/generate-greenplum-path.sh

```diff
@@ -56,26 +56,26 @@ if [ -h \${GPHOME}/../hawq ]; then
 fi
 EOF
 
-# OSX does not need JAVA_HOME 
+# OSX does NOT have DYLD_LIBRARY_PATH, add it
 if [ "${PLAT}" = "Darwin" ] ; then
-cat << EOF
-PATH=\$GPHOME/bin:\$GPHOME/ext/python/bin:\$PATH
-DYLD_LIBRARY_PATH=\$GPHOME/lib:\$GPHOME/ext/python/lib:\$DYLD_LIBRARY_PATH
+    cat << EOF
```

Comment:
remove spaces, it's confusing because the other lines in the if block don't have them.